### PR TITLE
Prepare for v1.6.0-rc1

### PR DESCRIPTION
**Pull request by marcopeereboom**
Add `gencerts` and install client certs in `politeiavoter` and copy the cert to `dcrwallet`.

### CODE REVIEW

File: cmd/dcrinstall/dcrinstall.go [hunk 1]
```diff
@@ -427,6 +433,34 @@ func (c *ctx) walletDBExists() bool {
...
+	// Create certificate for politeiavoter
+	gencertsExe := filepath.Join(c.s.Destination,
+		"decred-"+c.s.Tuple+"-"+version, "gencerts")
+	piDir := dcrutil.AppDataDir("politeiavoter", false)
+	piClientCert := filepath.Join(piDir, "client.pem")
+	o, err := exec.Command(gencertsExe, piClientCert,
+		filepath.Join(piDir, "client-key.pem")).CombinedOutput()
+	if err != nil {
+		panic(string(o))
+		return err
```
Comment (jrick) [inline cmd/dcrinstall/dcrinstall.go]: i would print this before returning the error, or wrap the output with the error

**Review by jrick — CHANGES_REQUESTED** [review]
only a small nit with the error handling

File: cmd/dcrinstall/dcrinstall.go (hunk 1 above)
Comment (marcopeereboom) [reply cmd/dcrinstall/dcrinstall.go]: Fixed.

File: cmd/dcrinstall/dcrinstall.go [hunk 2]
```diff
@@ -427,6 +433,34 @@ func (c *ctx) walletDBExists() bool {
...
+	return exist(filepath.Join(dir, filename))
+}
+
+func (c *ctx) generateClientCerts(version string) error {
+	// Create certificate for politeiavoter
+	gencertsExe := filepath.Join(c.s.Destination,
+		"decred-"+c.s.Tuple+"-"+version, "gencerts")
+	piDir := dcrutil.AppDataDir("politeiavoter", false)
+	piClientCert := filepath.Join(piDir, "client.pem")
+	o, err := exec.Command(gencertsExe, piClientCert,
```
Comment (jrick) [inline cmd/dcrinstall/dcrinstall.go]: this might not work on windows

File: cmd/dcrinstall/bitcoin.go [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+# Pre-generate this many public/private key pairs, so wallet backups will be valid for
+# both prior transactions and several dozen future transactions.
+#keypool=100
+
+# Enable pruning to reduce storage requirements by deleting old blocks.
+# This mode is incompatible with -txindex and -rescan.
+# 0 = default (no pruning).
+# 1 = allows manual pruning via RPC.
+# >=550 = target to stay under in MiB.
+#prune=550
```
Comment (buck54321) [inline cmd/dcrinstall/bitcoin.go:154]: I think the discussion on Matrix has been that we'll have pruning on by default.

Comment (marcopeereboom) [reply cmd/dcrinstall/bitcoin.go:154]: These are the droids you are looking for: https://github.com/decred/decred-release/pull/180/files#diff-607d78322296df85cc3b08b83fb89e58f2cad7755682c429ab145e7aea7e346eR602-R610

This is simply a sample bitcoind.conf file that we mangle on the way out.

File: cmd/dcrinstall/dcrinstall.go (hunk 2 above)
Comment (marcopeereboom) [reply cmd/dcrinstall/dcrinstall.go]: Fixed.

**Review by jrick — APPROVED** [review]
tACK on windows

**Pull request merged**